pytest_operator/plugin.py

```diff
@@ -406,8 +414,17 @@ async def _setup_model(self):
             await self._controller.connect(self.controller_name)
             on_cloud = f" on cloud {self.cloud_name}" if self.cloud_name else ""
             log.info(f"Adding model {self.model_full_name}{on_cloud}")
+
+            model_config_file = Path(self.model_config)
+            model_config = None
+            if not model_config_file.exists():
+                log.warning("Can't apply model-config from %s", model_config_file)
```

Comment:
If the user has specified a `--model-config` file, but the file couldn't be found, they will probably want to know about it. I would go further and log this message at the error level, or maybe even halt execution by raising an Exception.

If the user has *not* specified a `--model-config` file, it seems better to just not try to load it. No need for a warning in that case.

Reply:
agree, things always make more sense after a day.  great feedback.  I'll update to this